starter-code/main.js

```diff
@@ -53,7 +60,156 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
+          case "4":
+            // 4.- List by name all companies founded in february of 2004.
```

Comment:
Great! Pero las comillas no son necesarias:

`{ $and: [{ founded_year: 2004 }, { founded_month: 2 }] }`